lib/npg_pipeline/function/warehouse_archiver.pm

```diff
@@ -120,16 +117,11 @@ sub _update_warehouse_command {
       }
     }
 
-    $d = npg_pipeline::function::definition->new(
-      created_by => __PACKAGE__,
-      created_on => $self->timestamp(),
-      identifier => $self->label,
+    $d = $self->create_definition({
       command    => $command,
       num_cpus   => [0],
```

Comment:
Number of CPUs definition has to be moved to the definition of the function resources